{{cookiecutter.project_slug}}/requirements/dev-requirements.in

```diff
@@ -6,3 +6,4 @@ flake8-docstrings
 flake8-bugbear
 isort
 safety
+ipython
```

Comment:
I guess these depend on our decision whether development dependencies should be in production images. I can see both pros and cons.

Reply:
The scope of this change is about whether ipython should be included among the dev dependencies  - not whether dev dependencies should be part of the prod images or not, I think that's a concern that should be raised separately. There's a PR for it [here](https://github.com/DD-DeCaF/cookiecutter-flask-microservice/pull/95) which seems to have gone stale, so I wanted to bring it up again.

Reply:
Okay, but again. They are all part of the base-image anyway. So it's not required any longer I think.